docs/versionpicker.html

```diff
@@ -0,0 +1,115 @@
+<!-- Start interactive version picker -->
+<div id="installation" v-cloak>
+  <header class="row">
+    <h3>Paste this into the <code>&lt;head&gt;</code> of your HTML:</h3>
+    <button type="button" @click="copyToClipboard">
+      <transition>
+        <span v-if="copyStatus === 'success'" class="emoji">✔</span>
+        <span v-else-if="copyStatus === 'failed'" class="emoji">❌</span>
+      </transition>
+      Copy to clipboard
+    </button>
+  </header>
+
+  <transition mode="out-in">
+    <pre :key="selectedVersion.fileSnippet"><code>{{selectedVersion.fileSnippet}}</code></pre>
+  </transition>
+
+  <h3>Version options:</h3>
+  <form action="./" method="post" @submit.prevent>
```

Comment:
Can we hide the advanced options under a `<details>` element?

Reply:
Done, dark/light selection is always visible, the rest is in a `<details>` element "Additional options".
We just need to be careful when we merge #87 and ensure that our docs site works with the new details styling proposed there.